api/envoy/api/v2/listener/udp_listener_config.proto

```diff
@@ -30,6 +30,17 @@ message UdpListenerConfig {
 
     google.protobuf.Any typed_config = 3;
   }
+  
+  enum UdpShutDownMode {
```

Comment:
cc @alyssawilk @danzh2010 @mattklein123 this is initial draft for config layout.